### PR TITLE
Move accordion behind auto filter feature switch

### PR DESCRIPTION
**Pull request by silvacb**
## What does this change?
Moves the Live Feed header accordion behind the auto filter AB test. 
**This test is currently switched off and set to 0%.**

## Why?
As part of the automatic filters redesign, the header accordion is being removed.
NB: When removing the accordion, we lose horizontal padding. In the interest of keeping the PR small, this has been ticketed here (https://trello.com/c/XL3ulWuM/414-auto-filters-fix-padding-in-live-feed) and will be addressed in an upcoming PR.

## Screenshots

**Before:** 
Screenshot 2022-06-22 at 11 34 43

**After:** 
Screenshot 2022-06-22 at 11 34 58

### CODE REVIEW

File: dotcom-rendering/src/web/layouts/LiveLayout.tsx [hunk 1]
```diff
@@ -301,6 +301,9 @@ export const LiveLayout = ({ CAPIArticle, NAV, format }: Props) => {
 	const showKeyEventsCarousel =
 		CAPIArticle.config.abTests.keyEventsCarouselVariant == 'variant';
 
+	const isInFilteringBeta =
+		CAPIArticle.config.abTests.automaticFiltersVariant == 'variant';
```
Comment (marjisound) [inline dotcom-rendering/src/web/layouts/LiveLayout.tsx:305]: Maybe this needs testing, but I think we could have scenarios where `topics` is not provided from frontend meaning we don't want to add filter for this blog, but the `automaticFiltersVariant` could still be `variant`

Comment (silvacb) [reply dotcom-rendering/src/web/layouts/LiveLayout.tsx]: Thanks for the suggestion Marjan, this has been implemented now by adding an additional check for topics.

File: dotcom-rendering/src/web/layouts/LiveLayout.tsx [hunk 2]
```diff
@@ -301,6 +301,10 @@ export const LiveLayout = ({ CAPIArticle, NAV, format }: Props) => {
 	const showKeyEventsCarousel =
 		CAPIArticle.config.abTests.keyEventsCarouselVariant == 'variant';
 
+	const isInFilteringBeta =
+		CAPIArticle.config.abTests.automaticFiltersVariant == 'variant' &&
```
Comment (joecowton1) [inline dotcom-rendering/src/web/layouts/LiveLayout.tsx:305]:
```suggestion
		CAPIArticle.config.switches.automaticFilters &&
```

**Review by marjisound — APPROVED** [review]
Great work Silvija 👍

**Pull request merged**